odpf/shield/v1beta1/admin.proto

```diff
@@ -0,0 +1,148 @@
+syntax = "proto3";
+
+package odpf.shield.v1beta1;
+
+import "google/api/annotations.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+import "validate/validate.proto";
+import "odpf/shield/v1beta1/models.proto";
+
+option go_package = "github.com/odpf/proton/shield/v1beta1;shieldv1beta1";
+option java_outer_classname = "Shield";
+option java_package = "io.odpf.proton.shield.v1beta1";
+
+// These annotations are used when generating the OpenAPI file.
+option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+  info: {
+    title: "Shield Administration API",
+    version: "0.2.0";
+  };
+  schemes: HTTP;
+};
+service AdminService {
+  // Users
+  rpc ListAllUsers(ListAllUsersRequest) returns (ListAllUsersResponse) {
+    option (google.api.http) = {
+      get: "/v1beta1/users"
```

Comment:
@kushsharma shouldn't admin also come in this prefix path? Also any reason to move `v1beta1` back to path?